quickstarts/gigamon-appinsights/config.yml

```
@@ -0,0 +1,46 @@
+
+
+slug: gigamon-appinsights
+
+title: Gigamon Application Insights
+
+description: |
+  ## Deep Application Visibility
+  Analytics tools are only as smart as the data they receive. Gigamon Application Metadata Intelligence empowers 
+  New Relic with critical metadata attributes across thousands of business, consumer and IT applications and services. 
+  Get deep application visibility to rapidly pinpoint performance bottlenecks, quality issues and potential network security risks.
+
+summary: |
+  Powered by Gigamon's Application Metadata Intelligence (AMI), part of Application Intelligence, to help you monitor and manage complex digital initiatives.
+
+level: Verified
+
+authors:
+  - Gigamon
+
+keywords:
+  - gigamon
+  - application
+  - network
+  - AMI
+  - intelligence
+  - metadata
+  - siem
```

Then we can add those keywords over here
```suggestion
  - intelligence
  - metadata
  - siem
  - newrelic partner
  - NR1_addData
  - NR1_sys
```